blockchain/backup/NFTtheWorld.sol

```diff
@@ -13,46 +15,66 @@ contract NFTtheWorld {
     uint256[] public dropHashes;
 
     struct NFTOwnership {
-        address owner;
-        uint256 nftId;
+        address payable owner;
+        string uri;
         uint256 dropId;
         uint256 dropTime;
         address reservedFor;
+        uint256 weiPrice;
```

Comment:
Not a pronounceable variable name, see review guidelines. (Besides I'm not sure what it should mean.)

Reply:
This is a very straightforward name (https://www.investopedia.com/terms/w/wei.asp) and I strongly recommend keeping it this way since it describes exactly what this variable is

Reply:
Sorry, yes. Very legitimate. Problem of knowledge on my side.